sentry_kube/cli/get_image.py

```diff
@@ -0,0 +1,41 @@
+import click
+from libsentrykube.service import get_deployment_image
+
+__all__ = ("get_image",)
+
+
+SERVICE_DEPLOYMENT_CONTAINER_MAP = {
+    "chartcuterie": ("chartcuterie-web-production", "chartcuterie"),
+    "getsentry": ("getsentry-web-default-common-production", "sentry"),
+    "getsentry-control": ("getsentry-control-web-default-common-production", "sentry"),
+    "relay": ("relay-default-production", "relay"),
+    "relay-pop": ("relay-pop", "relay"),
+    "release-registry": ("release-registry", "release-registry"),
+    "reload": ("reload", "reload"),
+    "script-runner": ("script-runner/script-runner-region", "script-runner-region"),
+    "seer": ("seer-web-severity", "seer"),
+    "seer-gpu": ("seer-gpu-web-group-ingest", "seer-gpu"),
+    "snuba": ("snuba-api-production", "api"),
+    "super-big-consumers": ("sbc-arroyo-errors", "super-big-consumers"),
+    "symbol-collector": ("symbol-collector", "symbol-collector"),
+    "tempest": ("tempest-default", "tempest"),
+    "vroom": ("vroom-default-production", "vroom"),
+}
```

Comment:
This is configuration data, we should be pulling it in from `ops` config rather than including it here.

Reply:
https://github.com/getsentry/ops/pull/16219